### ISSUE TITLE
Vacuous Truth

### ISSUE THREAD

**Issue by Andreas-Oian** [jfremstad/matematisk_ordliste#172]
**Forslag:**
bokmål,nynorsk,engelsk
tom sannhet,tom sanning,vacuous truth
tomlig sann,tomleg sann,vacuously true

**Kommentar:**
Eg held på å lære meg bevisteknikkar og kom over omgrepet "vacuous truth." Eg fann ikkje omgrepet i ordlistane, men eg fann eit [forum](https://dict.leo.org/forum/viewUnsolvedquery.php?idForum=2&idThread=1372071&lp=ende&lang=en) med nokon tyskarar som stussa på nett det same som meg. På tysk er visst omgrepet "leere wahreit", som direkte omsett tyder "tom sanning". Eg er litt meir i stuss når det kjem til adjektiv-forma av ordet, men "tomleg sann" var det beste eg kom på.

**Comment by fredrik-bakke** [comment]
Hei Andreas, takk for forslaget!
Det later ikke til at adverbsformen "tomlig" er etablert. F.eks. gir den [ingen resultater på ordbokene.no](https://ordbokene.no/bm,nn/search?q=tomlig&scope=ei). Har du vurdert om begrepene "triviell sannhet" og "trivielt sann" fungerer i din kontekst? Jeg er også skeptisk til om "tom sannhet" er en god oversettelse for "vacuous truth", det er jo ikke sannheten som er tom, men måten den eksisterer på.

**Comment by Andreas-Oian** [comment]
Eit utsegn er ein "vacuous truth", viss det er på forma $\forall x \in A, P(x)$ og $A$ er ei tom mengde. "Alle troll kan køyre bil" er eit sant utsegn, sidan det ikkje finst troll. Eg får ikkje heilt kjensla av at "triviell sannhet" fungerer. Ein triviell sanning verkar meir som ein sanning som er ei direkte følgje av ein definisjon e.l. "Alle hjul er sirkelforma", er ein triviell sanning. 
[Leg-suffiksen](https://ordbokene.no/bm,nn/search?q=-leg&scope=ei) tyder at noko liknar på eit eller anna. Eg føler at "tomleg" gir meining, men det er vel ikkje berre berre å "lage" eit nytt ord.

**Comment by wishfort36** [comment]
**Forslag**:

bokmål, nynorsk, engelsk
intetsigende sannhet, intetsigande(?) sannhet, vacuous truth

Jeg bruker denne selv. Slik jeg tolker Merriam-Webster og Wiktionary bør intetsigende være en god oversettelse av "vacuous".

**Comment by Andreas-Oian** [comment]
Ja, det er ei god omsetjing! På nynorsk ville ein sagt "inkjeseiande sanning".
